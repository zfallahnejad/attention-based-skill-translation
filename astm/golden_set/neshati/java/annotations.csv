438154,annotations
571407,annotations
203907,annotations
103154,annotations
70604,annotations
21234,annotations
383861,annotations
342852,annotations
897024,annotations
573032,annotations
22656,annotations
605744,annotations
680925,annotations
131368,annotations
157882,annotations
204788,annotations
