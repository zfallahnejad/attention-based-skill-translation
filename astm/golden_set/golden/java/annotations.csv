438154,56,22,0.717948717948718
571407,36,21,0.631578947368421
203907,34,36,0.4857142857142857
103154,32,10,0.7619047619047619
70604,26,27,0.49056603773584906
21234,26,26,0.5
383861,25,17,0.5952380952380952
342852,23,23,0.5
897024,23,13,0.6388888888888888
573032,18,12,0.6
22656,14,7,0.6666666666666666
680925,13,3,0.8125
605744,13,5,0.7222222222222222
131368,12,4,0.75
157882,11,6,0.6470588235294118
204788,11,15,0.4230769230769231
